apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -34,6 +34,14 @@ export const MB_SENSOR_VARS = [
   'compass'
 ];
 
+// milliseconds between samples for sensors
+export const SAMPLE_INTERVAL = 50;
+
+// Max number of milliseconds for which we store historical data
+export const MAX_SENSOR_BUFFER_LENGTH = 3000;
```

Comment:
Ah, just realised it's duration, rather than buffer length, so could perhaps we could name accordingly?